githubcollective/config.py

```diff
@@ -303,7 +315,11 @@ def _get_teams(self):
            not self._github['teams']:
             self._github['teams'] = {}
             for item in self.github._gh_org_teams():
-                if not item['name'].startswith(TEAM_PREFIX):
+                # TODO: whats happening here? This seems to rely on
+                # team prefix being used.  Skips evyerthing else.
+                # If we are taking the no-prefix support approach
+                # then the script should accept all groups, ya?
+                if not item['name'].startswith(self.team_prefix):
```

Comment:
I think that simply having this behave like this would address the issue... but definitely want some feedback.

```
    if self.team_prefix and not item['name'].startswith(self.team_prefix):
```